ethsigner/core/src/integration-test/java/tech/pegasys/ethsigner/jsonrpcproxy/IntegrationTestBase.java

```diff
@@ -236,83 +234,83 @@ void sendPostRequestAndVerifyResponse(
 
   void sendPostRequestAndVerifyResponse(
       final EthSignerRequest request, final EthSignerResponse expectResponse, final String path) {
-    given()
-        .when()
-        .body(request.getBody())
-        .headers(request.getHeaders())
-        .post(path)
-        .then()
-        .statusCode(expectResponse.getStatusCode())
-        .body(equalTo(expectResponse.getBody()))
-        .headers(expectResponse.getHeaders());
+
+    final Response response =
+        given()
+            .when()
+            .body(request.getBody())
+            .headers(RestAssuredConverter.headers(request.getHeaders()))
+            .post(path);
+
+    assertThat(response.statusCode()).isEqualTo(expectResponse.getStatusCode());
+    assertThat(response.headers())
```

Comment:
We should also verify that body is what we expected for all these send and verify methods

Reply:
agreed